src/utilities/dialogs/Content.tsx

```diff
@@ -0,0 +1,7 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { ReactElement, ReactNode } from 'react';
+
+// eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
This is different sytax to the Trigger but they do the same thing. Maybe just use the same as the Trigger and remove the es-lint rule

Reply:
It's not quite the same. in the Trigger, children is one ReactElement but Content receive a ReactNode. You would get an error. I don't want people to wrap their content in a container to use the component so I wrap it in a Fragment. I already checked that and disabling the es-lint rule here is the cleanest and best solution to keep the ease of use.